criu/cr-restore.c

```diff
@@ -2394,6 +2391,9 @@ int cr_restore_tasks(void)
 	if (fdstore_init())
 		goto err;
```

Comment:
it seems that goto err should be used only after cr_plugin_init, because it calls cr_plugin_fini.